angular-workspace/projects/ni/nimble-angular/table-column/date-text/nimble-table-column-date-text.directive.ts

```diff
@@ -2,7 +2,7 @@ import { Directive, ElementRef, Input, Renderer2 } from '@angular/core';
 import { type TableColumnDateText, tableColumnDateTextTag } from '@ni/nimble-components/dist/esm/table-column/date-text';
 import { BooleanValueOrAttribute, NumberValueOrAttribute, toBooleanProperty, toNullableNumberProperty } from '@ni/nimble-angular/internal-utilities';
 import { NimbleTableColumnBaseDirective } from '@ni/nimble-angular/table-column';
-import type {
+import {
```

Comment:
```suggestion
export {
```

just use the re-export syntax https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#re-exporting_aggregating

and delete all the single line exports added below

Reply:
I can't because I'm actually using those types as well as re-exporting them.